### PR TITLE
Changed build target to .NET Standard 2.0 🚀

### PR DESCRIPTION
**Pull request by sven-n**
Currently, this project targets the .NET Framework. However, we want to migrate to .NET 5 and beyond.

I analyzed the code and found no issues when it targets .NET Standard 2.0, which is compatible to both .NET Framework and newer .NET versions.

If you have access to the published nuget package (#10, @bhattvishal ?), it would be great to get an updated package as well.

Thank you 😀

### CODE REVIEW

**Comment by kallemooo** [comment]
The automatic build system did not like the commit.
Can you please check the error?

For the nuget package it is an unofficial upload, that I did not do. I will try again to get the rights to replace that version,

**Comment by sven-n** [comment]
The target name of the pre-build had to be changed. It should work now.

**Comment by sven-n** [comment]
Hi, any news? You can re-run the build check, btw ;-)

File: Asap2/Asap2.csproj [hunk 1]
```diff
@@ -64,70 +35,33 @@
...
-    <Reference Include="System.Xml" />
   </ItemGroup>
+
+
+  <PropertyGroup>
+    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+  </PropertyGroup>
+
   <ItemGroup>
-    <Compile Include="Asap2Tree\Asap2Base.cs" />
```
Comment (kallemooo) [inline Asap2/Asap2.csproj:83]: Why are all project files not included anymore?

**Comment by kallemooo** [comment]
> Hi, any news? You can re-run the build check, btw ;-)

The change causes the build to fail.
Please check this.

Did you build using the command line tools on your computer as the CI build does?

I cannot check this as I do not have windows installed any longer.

File: Asap2/Asap2.csproj (hunk 1 above)
Comment (sven-n) [reply Asap2/Asap2.csproj:83]: I had to change the project file format to the new SDK format. With this format, all files (including subfolders) are included by convention by default.
More info here: https://learn.microsoft.com/en-us/dotnet/core/project-sdk/overview

**Comment by sven-n** [comment]
Oh, didn't expect it still fails :-( I built it on my machine using vs 2019. I'll check it with the command line when I'm back at the office.

**Pull request closed without merging**